src/liballoc/linked_list.rs

```diff
@@ -1071,6 +1071,15 @@ impl<'a, T, F> Iterator for DrainFilter<'a, T, F>
     }
 }
 
+#[unstable(feature = "drain_filter", reason = "recently added", issue = "43244")]
```

Comment:
Trait impls are always stable, so this'll need to be `#[stable(feature = "drain_filter_drop", since = "1.25.0")]`